app/services/solidus_pay_tomorrow/handle_order_creation_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  class HandleOrderCreationService < SolidusPayTomorrow::BaseService
+    attr_reader :order, :payment_method
+
+    def initialize(order:, payment_method: )
+      @order = order
+      @payment_method = payment_method
+      super
+    end
+
+    def call
+      @result = create_order
+      create_payment
+      @result
+    end
+
+    private
+
+    def create_order
```

Comment:
NIT: I think that create_order_on_pay_tomorrow will be a better name here

Reply:
Done in de3d582fc3ea85827161f935a1388d6ae6f5ade4